doc/classes/HingeJoint3D.xml

```diff
@@ -53,7 +53,7 @@
 		<member name="angular_limit/relaxation" type="float" setter="set_param" getter="get_param" default="1.0">
 			The lower this value, the more the rotation gets slowed down.
 		</member>
-		<member name="angular_limit/softness" type="float" setter="set_param" getter="get_param" default="0.9">
+		<member name="angular_limit/softness" type="float" setter="set_param" getter="get_param" default="0.9" deprecated="This property is never set by the engine and is kept for compatibility purpose.">
```

Comment:
If you're doing deprecation you shouldn't delete the binds

Reply:
What's the best approach? Remove the deprecation tags and leave it as is or revert everything and just deprecate the parameter in the documentation?

Reply:
Just deprecate it in the documentation, deprecation means the code remains intact here, so if you want to go with that route you should just have the documentation changes added